Include/internal/pycore_optimizer.h

```diff
@@ -73,6 +73,7 @@ typedef struct _PyExecutorObject {
     uint32_t exit_count;
     uint32_t code_size;
     size_t jit_size;
+    uint32_t run_count;
```

Comment:
Since this is effectively a boolean field, can you make it part of the bitfield on the `_PyVMData` struct (right after `valid` and `linked`? You can steal a bit from `chain_depth`, since it doesn't need 14. Then we can treat the field like a boolean everywhere.

Reply:
Ha! That's very true, since we ended up with a threshold of 1. Very clever and some nice cleanup after that experimentation